test/integration/activate_int_test.go

```diff
@@ -54,11 +54,7 @@ func (suite *ActivateIntegrationTestSuite) TestActivateWithoutRuntime() {
 	close := suite.addForegroundSvc(ts)
 	defer close()
 
-	cp := ts.SpawnWithOpts(
-		e2e.OptArgs("activate", "ActiveState-CLI/Empty"),
-		e2e.OptAppendEnv(constants.DisableRuntime+"=true"),
-	)
-	cp.Expect("Skipping runtime setup")
+	cp := ts.Spawn("activate", "ActiveState-CLI/Empty")
```

Comment:
Set async runtime?

Reply:
No, this is the empty project and we don't care about it. This use of DisableRuntime is vestigal.